0-bootstrap/README-Jenkins.md

```diff
@@ -105,34 +124,27 @@ You arrived to these instructions because you are using the `jenkins_bootstrap`
     cp -R ../terraform-example-foundation/0-bootstrap/* .
```

Comment:
This line should be like the other ones like 

    cp -RT ../terraform-example-foundation/0-bootstrap/ .

Reply:
Done.